pubspec.yaml

```
@@ -4,7 +4,7 @@ description: Checks for missing, under-promoted, over-promoted, and unused depen
 homepage: https://github.com/Workiva/dependency_validator
 
 environment:
-  sdk: '>=2.19.0 <3.0.0'
+  sdk: '^3.0.0'
```

Could we update this to still allow for dart v2? We still have a few consumers of dependency_validator which have yet to migrate

```suggestion
  sdk: '>=2.19.0 <4.0.0'
```

---

That's due to pubspec_parse, which, since the 1.0 that you were using before, has required more and more recent versions of Dart. In fact, version 1.5 [will require Dart 3.6](https://github.com/dart-lang/tools/pull/1946). 

This shouldn't be an issue though, since Dart [treats versions after 2.12 to be compatible with 3.0](https://dart.dev/resources/dart-3-migration#dart-3-backwards-compatibility). The only things that would cause a breakage from 2.19 are (list in the link above as "unversioned changes"): 
- packages that don't have null safety, or use dependencies that don't have null safety. Given that null safety came out 4 years ago and almost all packages one would depend on have migrated as well, I'd think it makes sense to no longer support pre-null-safe code in future updates, and if customers still don't have it, they can continue to use the current version of dependency_validator. In other words, customers without sound null safety are missing out on _many_ more important features than just this tool.
- Some APIs that had already been deprecated for years were removed
- Some tools were renamed (eg, `stagehand` -> `dart create`)
- Some minor other changes, see the link above